saos-webapp/src/main/webapp/WEB-INF/view/search/judgmentSearchForm.jsp

```diff
@@ -22,13 +22,86 @@
 		  <spring:message var="searchFormHintTitle" code="judgmentSearch.hint.searchForm.title" />
 		  <spring:message var="searchFormHintContent" code="judgmentSearch.hint.searchForm.content" arguments="${searchFormHintAddress}" />
 
-		
 		  <saos:hint content="${searchFormHintContent}" title="${searchFormHintTitle}" placement="left" ></saos:hint>
 		</div>
 		
-		<saos:formFieldText path="all" labelName="input-search-all" labelText="judgmentSearch.formField.all" />
+		
+		<div class="form-section-container" >
```

Comment:
As we agreed yesterday the whole court info-form section should be in a separate jsp file, so it could be easily included on analysis page.